project-base/storefront/pages/feedback.tsx

```diff
@@ -0,0 +1,54 @@
+import { LoaderWithOverlay } from 'components/Basic/Loader/LoaderWithOverlay';
+import { Button } from 'components/Forms/Button/Button';
+import { CommonLayout } from 'components/Layout/CommonLayout';
+import { Webline } from 'components/Layout/Webline/Webline';
+import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { useCookiesStore } from 'store/useCookiesStore';
+import { getServerSidePropsWrapper } from 'utils/serverSide/getServerSidePropsWrapper';
+import { ServerSidePropsType, initServerSideProps } from 'utils/serverSide/initServerSideProps';
+import { showSuccessMessage } from 'utils/toasts/showSuccessMessage';
+
+const Feedback: FC<ServerSidePropsType> = () => {
+    const { t } = useTranslation();
+    const setCookiesStoreState = useCookiesStore((state) => state.setCookiesStoreState);
+    const isUserSnapEnabled = useCookiesStore((store) => store.isUserSnapEnabled);
+    const router = useRouter();
+    const [isLoadingOverlayVisible, setIsLoadingOverlayVisible] = useState(false);
+
+    const handleTurnOffUserSnap = () => {
+        if (isUserSnapEnabled) {
+            setIsLoadingOverlayVisible(true);
+            showSuccessMessage(t('The feedback tool has been successfully deactivated'));
```

Comment:
From the page and texts, it's not clear that the user enables this tool only for himself. Maybe it is worth mentioning?